reference/best_practises/common_best_practises.rst

```diff
@@ -0,0 +1,26 @@
+Common Best Practises
```

Comment:
@rybus Both seems possible and we already use "practises" everywhere in the documentation... I prefer with a "c" too :(
"Common" because there is 3 other parts in the best practices (coming soon:p).
- How to develop a reusable bundle/component/library
- How to do a PIM project (integrators)
- How to develop as a core developer (very close to reusable components)